search/text_search.cpp

```diff
@@ -12,7 +13,22 @@
 
 /** Main function
  */
+
+std::string lower(std::string word) {
+    int length = word.length();
+    std::string lc = "";
+
+    for (int i = 0; i < length; i++) {
+        lc += tolower(word[i]);
+    }
+
+    return lc;
+}
+
+void test() { assert(lower("abcd").compare("abcd") == 0); }

```

Comment:
Feel free to add more assert tests as well. 🙂
```suggestion
/**
 * @brief Self-test implementations
 * @returns void
 */
static void test() {
    assert(lower("abcd").compare("abcd") == 0);
}
```

Reply:
Yeah I'll soon add this

Reply:
Uh well... I didn't find much of test cases. Can you please suggest some? 😅